lib/new_relic/agent/instrumentation/kinesis/instrumentation.rb

```diff
@@ -0,0 +1,71 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module Kinesis
+    INSTRUMENTED_METHODS = %w[
```

Comment:
We need to update the instrumentation to create message broker spans for: 
`get_records`
`put_record`
`put_records`